rmw_connext_cpp/src/functions.cpp

```diff
@@ -1,3 +1,5 @@
+#include <rmw/types.h>
```

Comment:
We should move this include down to be grouped with the other ones from the same package.